src/main/webapp/definitions/alfresco-search-sql.yaml

```diff
@@ -0,0 +1,316 @@
+
+
+definitions:
+
+  ## Standard definition
+  Error:
+    type: object
+    required:
+      - error
+    properties:
+      error:
+        type: object
+        required:
+          - statusCode
+          - briefSummary
+          - stackTrace
+          - descriptionURL
+        properties:
+          errorKey:
+            type: string
+          statusCode:
+            type: integer
+            format: int32
+          briefSummary:
+            type: string
+          stackTrace:
+            type: string
+          descriptionURL:
+            type: string
+          logId:
+            type: string
+
+  SQLSearchRequest:
+    type: object
+    properties:
+      stmt:
+        type: string
+      format:
+        type: string
+      locales:
+        type: array
+        items:
+          type: string
+      timezone:
+        type: string
+      includeMetadata:
+        type: boolean
+
+
+  # ReponseQuery
+
+  SQLResultSetPaging:
+    description: Query results
+    type: object
+    properties:
+      list:
+        type: object
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/SQLResultSetRowEntry'
+  Pagination:
+    type: object
+    required:
+      - count
+      - hasMoreItems
+      - skipCount
+      - maxItems
+    properties:
+      count:
+        type: integer
+        format: int64
+        description: |
+          The number of objects in the entries array.
+      hasMoreItems:
+        type: boolean
+        description: |
+          A boolean value which is **true** if there are more entities in the collection
+          beyond those in this response. A true value means a request with a larger value
+          for the **skipCount** or the **maxItems** parameter will return more entities.
+      totalItems:
+        type: integer
+        format: int64
+        description: |
+          An integer describing the total number of entities in the collection.
+          The API might not be able to determine this value,
+          in which case this property will not be present.
+      skipCount:
+        type: integer
+        format: int64
+        description: |
+          An integer describing how many entities exist in the collection before
+          those included in this list.
+      maxItems:
+        type: integer
+        format: int64
+        description: |
+          The value of the **maxItems** parameter used to generate this list,
+          or if there was no **maxItems** parameter the default value is 100
+
+  SQLResultSetRowEntry:
+    type: object
+    properties:
+      label:
+        type: string
+      value:
+        type: string
+
+  SolrResultSet:
+    description: SQL results in solr formatting
+    type: object
+    properties:
+      result-set:
+        type: object
+        $ref: '#/definitions/docs'
+  docs:
+    type: array
```

Comment:
Should we add the description here and maybe use a more specific value than `doc` or can we assume it'll be clear for users?